.github/workflows/gradle.yml

```diff
@@ -84,9 +84,20 @@ jobs:
       with:
         fail_ci_if_error: true
 
+    - name: Cache Integration tests trends
+      if: github.ref == 'refs/heads/master' && matrix.platform == 'ubuntu-latest' && always()
+      uses: actions/cache@v1
+      with:
+        path: vividus-tests/output/integration-trends
```

Comment:
what about using single cache for all trends: `vividus-tests/output/history`?
and have a separation inside the directory: 
- `vividus-tests/output/history/integration-tests`
- `vividus-tests/output/history/system-tests`